test/functional/host_test.rb

```diff
@@ -219,3 +219,29 @@
     result.exit_code.must_equal HammerCLI::EX_OK
   end
 end
+describe 'disassociate host from vm' do
+  let(:cmd) { ["host","disassociate"] }
+
+  it 'successful disassociate host' do
+    params = ['--id=1']
+    expected_result = success_result("The host has been disassociated from VM\n")
+    api_expects(:hosts, :disassociate, 'disassociate hosts') do |params|
+      params['id'] == "1"
+    end.returns()
+
+    result = run_cmd(cmd + params)
+    assert_cmd(expected_result, result)
+  end
+
+  it 'prints error on missing host id' do
+    params = []
+    expected_result = CommandExpectation.new
+    expected_result.expected_err = [
+      "Failed to disassociated host from VM:",
+      "  Missing arguments for 'id'", ''].join("\n")
```

Comment:
Nitpick: Could you please put `''` on a new line so that it's more obvious we're expecting one more new line here?